### PR TITLE
Instructions Screen Design Implementation

### PR DESCRIPTION
**Pull request by tobitech**
Story: https://app.shortcut.com/smileid/story/xxx

## Summary

A few sentences/bullet points about the changes

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: SmileIDUI/Classes/SmileIDInstructionsScreen.swift [hunk 1]
```diff
@@ -19,22 +30,89 @@ public struct SmileIDInstructionsScreen<ContinueButton: View, CancelButton: View
...
+            .font(theme.typography.pageHeading.weight(.bold))
+            .foregroundColor(color(theme.colors.titleText))
+
+          Text("Your selfie is encrypted and used only for verification.")
+            .font(theme.typography.subHeading)
+            .foregroundColor(color(theme.colors.cardText))
+
+          // Rule cards
+          InstructionCard(
+            icon: Image(systemName: "square.stack.3d.up"),
```
Comment (liquidblack) [inline SmileIDUI/Classes/SmileIDInstructionsScreen.swift:80]: Proposal: https://www.svgrepo.com/svg/437286/sun-max
